adhocracy4/polls/api.py

```diff
@@ -93,6 +113,10 @@ def vote(self, request, pk):
                             self.request.data['agreed_terms_of_use']
                     }
                 )
+            else:
+                raise ValidationError({
```

Comment:
For the poll, it does not matter which error, I use, it always gives "Your answer could not be saved. Please check the data you entered again.". I will add an issue for that, but would not like to handle that in this PR.